cmd/codegen/util/commands.go

```diff
@@ -168,6 +168,8 @@ func (c *Command) RunWithoutRetry() (string, error) {
 	var r string
 	var e error
 
+	AppLogger().Debugf("Running %s %s %s", JoinMap(c.Env, " ", "="), c.Name, strings.Join(c.Args, " "))
```

Comment:
Depending on what you are after you can print a map via `%+v`

Reply:
Wanted to print the map as though it were environment variables.